workspaces/cli-server/src/spectacle/index.ts

```diff
@@ -197,10 +198,23 @@ export class LocalCliCapturesService implements IOpticCapturesService {
     };
   }
 
-  loadInteraction(
-    captureId: string,
-    pointer: string
-  ): Promise<IHttpInteraction> {
+  async getCaptureStatus(captureId: string): Promise<GetCaptureStatusResponse> {
```

Comment:
Well, this code is in the server, and it seemed sketchy to have the server try and make a request to itself. It could probably be abstracted into one place still though..

Reply:
Ah I missed that it's in the cli spectacle instance - yeah it's probably fine to duplicate this for now